pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -327,5 +240,6 @@ def target_field_should_be_first(x):
         return grouped_entries
 
     def get_entry(self, item):
-        item["comment"] = self.comment_dict.get(item["suggestion_id"])
-        return self.entry_class(ProxySubmission(item)).entry
+        item["suggestion_comment"] = self.comment_dict.get(
```

Comment:
because it was actually a suggestion_comment - and in the context of the class being more reusable i figured it was better to make that explicit